translations/translations.go

```diff
@@ -0,0 +1,47 @@
+package translations
+
+// TranslationMap converts from Japanese to English
+var TranslationMap = map[string]string{
+	"失敗":              "Failure",
```

Comment:
What is the CS of alignment here? Can we have one?

Reply:
What is CS in this context?

Reply:
How many spaces after the `:`? Are they align?

Reply:
14 spaces after the colon. If I attempt to straighten them out, it auto-corrects it for me on saving. I take it you don't like the misalignment?

Reply:
Yeah, it is usually aligned. Check here one example: https://golang.org/src/go/types/sizes.go

Reply:
I guess it is because is a string?